CONTRIBUTING.md

```diff
@@ -78,6 +78,30 @@ commit 2: add bar option
 If you are continuing the work of another person's PR and need to rebase/squash, please retain the
 attribution of the original author(s) and continue the work in subsequent commits.
 
+### Running tests
+
+To run the test suite on your local machine, you'll first need to do a little
+setup.
+
+If you're using macOS:
+
+```sh
+TRAVIS_OS_NAME=osx ./test/ci/before_install.sh
+```
+
+If you're using Linux:
```

Comment:
Also for Linux, that script is specifically for x86_64 architectures (since Travis only provides those machines). Not sure if it will error if you try to run it on an x86 architecture (mostly because I haven't tried it).